config/migrations/fractie-type-concept-scheme.sparql

```diff
@@ -0,0 +1,22 @@
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/application> {
+<http://data.vlaanderen.be/id/concept/Fractietype/Onafhankelijk> <http://mu.semte.ch/vocabularies/core/uuid> "fb2f9ed1-e8bc-432c-ac53-827566b863d9" ;
+	<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://mu.semte.ch/vocabularies/ext/Fractietype>, <http://www.w3.org/2004/02/skos/core#Concept> ;
+	<http://www.w3.org/2004/02/skos/core#inScheme> <http://data.vlaanderen.be/id/conceptscheme/Fractietype> ;
+	<http://www.w3.org/2004/02/skos/core#prefLabel> "Onafhankelijk" ;
+	<http://www.w3.org/2004/02/skos/core#topConceptOf> <http://data.vlaanderen.be/id/conceptscheme/Fractietype> .
+
+<http://data.vlaanderen.be/id/concept/Fractietype/Samenwerkingsverband> <http://mu.semte.ch/vocabularies/core/uuid> "260d6c7e-fbd4-4849-af26-e828001bc28d" ;
+	<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://mu.semte.ch/vocabularies/ext/Fractietype>, <http://www.w3.org/2004/02/skos/core#Concept> ;
+	<http://www.w3.org/2004/02/skos/core#inScheme> <http://data.vlaanderen.be/id/conceptscheme/Fractietype> ;
+	<http://www.w3.org/2004/02/skos/core#prefLabel> "Samenwerkingsverband" ;
+	<http://www.w3.org/2004/02/skos/core#topConceptOf> <http://data.vlaanderen.be/id/conceptscheme/Fractietype> .
+
+<http://data.vlaanderen.be/id/conceptscheme/Fractietype> <http://mu.semte.ch/vocabularies/core/uuid> "363ada78-3cc2-456f-a4f9-b01793bc9ef9" ;
+	<http://purl.org/dc/terms/modified> "2018-06-04"^^<http://www.w3.org/2001/XMLSchema#date> ;
+	<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#ConceptScheme> ;
+	<http://www.w3.org/2004/02/skos/core#note> "Type fractie waartoe een mandataris hoort." ;
+	<http://www.w3.org/2004/02/skos/core#prefLabel> "Fractietype" .
```

Comment:
Not only for this label but do we have a structure for this? Most of the time everything is just written in `
Pascal-case` for the `prefLabel`